settings/browsersettings.cpp

```diff
@@ -13,7 +13,9 @@
 #include <QXmlStreamReader>
 #include "browsersettings.h"
 
-static const QString openSearchPath("/usr/lib/mozembedlite/chrome/embedlite/content/");
+static const QStringList gOpenSearchPathList = QStringList() \
+                                            << QString("/usr/lib/mozembedlite/chrome/embedlite/content/") \
+                                            << QDir::homePath() + "/.local/share/org.sailfishos/sailfish-browser/opensearch/";
```

Comment:
```QStandardPaths::writableLocation(QStandardPaths::DataLocation) + "/searchEngines"``` will have different paths for browser application and browser settings.
For browser application it is ```/home/nemo/.local/share/jolla-settings/searchEngines```.
For browser settings it is ```/home/nemo/.local/share/org.sailfishos/sailfish-browser/searchEngines```.